packages/core/src/lib/collect-and-persist.unit.test.ts

```diff
@@ -25,6 +29,32 @@ vi.mock('./implementation/persist', () => ({
 }));
 
 describe('collectAndPersistReports', () => {
+  it('should normalize options internally (default values)', async () => {
+    const partialConfig: CollectAndPersistReportsOptions = {
+      plugins: MINIMAL_CONFIG_MOCK.plugins,
+      categories: MINIMAL_CONFIG_MOCK.categories,
+      verbose: false,
+      progress: false,
+    };
+    await collectAndPersistReports(partialConfig);
+
+    expect(collect).toHaveBeenCalledWith(partialConfig);
+
+    expect(persistReport).toHaveBeenCalledWith(
+      {
+        packageName: 'code-pushup',
+        version: '0.0.1',
+        date: expect.stringMatching(ISO_STRING_REGEXP),
+        duration: 0,
+        categories: [],
+        plugins: [],
+      },
+      normalizePersistConfig(partialConfig?.persist),
+    );
```

Comment:
The relevant assertion regarding persist defaults (i.e. the 2nd parameter) kinda gets lost in the surrouding noise (the 1st parameter takes the focus - 8 lines vs 1 line). It's better to be more explicit about the relevant parts and less explicit about what's irrelevant to this scenario.

```suggestion
    expect(persistReport).toHaveBeenCalledWith(
      expect.any(Object),
      {
        outputDir: PERSIST_OUTPUT_DIR,
        filename: PERSIST_FILENAME,
        format: PERSIST_FORMAT,
      },
    );
```

Reply:
signal/noice... THX!